tutorials/celestia-da.md

```diff
@@ -34,35 +34,41 @@ ignite scaffold chain gm --address-prefix gm --no-module
 Install the Rollkit app to ignite:
 
 ```bash
+cd $HOME/gm
 ignite app install github.com/ignite/apps/rollkit@rollkit/v0.2.0
 ```
 
-Next, move to the `gm` directory and add the Rollkit app:
+Add the Rollkit app:
 
 ```bash
-cd $HOME/gm
 ignite rollkit add
 ```
 
+Build the Rollkit chain:
+
+```bash
+ignite chain build
+```
+
 Initialize the Rollkit chain configuration:
 
 ```bash
 ignite rollkit init
 ```
 
-This will create a `$HOME/.gm` directory with the chain configuration files.
+This will create a `$HOME/.gm` directory with all the chain configuration files.
 
 
-## 🧰 Configuring your rollup {#configuring-your-rollup}
+## 🧰 Configuring your sovereign rollup {#configuring-your-sovereign-rollup}
```

Comment:
does this also change any links back to this section? i.e. any links to celestia-da.md#configuring-your-rollup need to now link to #configuring-your-sovereign-rollup

Reply:
@a26nine

Reply:
I don't see any backlinks to this section anywhere else.